noir-projects/aztec-nr/uint-note/src/uint_note.nr

```diff
@@ -50,9 +48,8 @@ impl NoteHash for UintNote {
         // perform both steps of the partial note computation.
 
         // First we create the partial note from a commitment to the private content (including storage slot).
-        let private_content = UintPartialNotePrivateContent { owner: self.owner };
```

Comment:
The `UintPartialNotePrivateContent` was now an empty struct so I dropped and now there is just a standalone `compute_partial_commitment` function.